eng/versioning/version_client.txt

```diff
@@ -102,7 +102,7 @@ com.azure:azure-core-test;1.27.0-beta.2;1.27.0-beta.3
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.50;1.0.0-beta.51
 com.azure:azure-core-tracing-opentelemetry-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-version-tests;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-cosmos;4.63.3;4.64.0-beta.1
+com.azure:azure-cosmos;4.63.2;4.64.0-beta.1
```

Comment:
Why this azure-cosmos version changed?

Reply:
Just to be consistent with the version in the sdk bom, changed only in this release, won't affect the main branch

Reply:
I guess @Netyyyy want to make sure the version shipped in our library aligns with the version defined in the Azure SDK BOM. @Netyyyy please make sure you won't downgrade the azure-cosmos version when merge back to the main branch.